guides/common/modules/proc_creating-new-ssh-keys-for-remote-execution.adoc

```diff
@@ -0,0 +1,19 @@
+[id="creating-new-ssh-keys-for-remote-execution_{context}"]
+= Creating new SSH keys for remote execution
+
+By default, {Project} uses a non-passphrase protected SSH key to execute remote jobs on hosts.
+You can create additional passphrase protected or non-passphrase protected SSH keys to authenticate for remote execution jobs on hosts.
+
+.Procedure
+. On your {ProjectServer} or {SmartProxyServer}, create a new SSH key:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# ssh-keygen
```

Comment:
Just generating the key is not enough. The key in questio has to be placed into an expected location (and thus replace the old key) so something like `ssh-keygen -f ~foreman-proxy/.ssh/id_rsa_foreman_proxy` would be appropriate

Alternatively, you can change the expected location by passing a combaination of `--foreman-proxy-plugin-remote-execution-script-ssh-identity-dir=$DIR` and `--foreman-proxy-plugin-remote-execution-script-ssh-identity-file=$KEY` to the installer